spec/Section 3 -- Type System.md

```diff
@@ -1973,6 +1973,9 @@ type Book @delegateField(name: "pageCount") @delegateField(name: "author") {
 extend type Book @delegateField(name: "index")
 ```
 
+Additionally, directive may be defined as exposed by including the "exposed" keyword.
+Making a directive exposed effects only its inclusion in "directiveAnnotations" list in introspection. Non-exposed directive are skipped from that list though they are not excluded from the "directives" list of the Schema.
```

Comment:
Making a directive exposed affects only its inclusion in the "directiveAnnotations" list in introspection. Non-exposed directives are skipped from that list though they are not excluded from the "directives" list of the Schema.